### PR TITLE
6 services section 21.03.2023

### PR DESCRIPTION
**Pull request by InnaLazarenko**

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -17,6 +18,7 @@ export default function Home() {
       <Main>
         <Hero />
         <CaseStudies data={caseStudies} />
+        <Services />
```
Comment (var-bin) [inline pages/index.tsx:21]: Pass data here

**Review by var-bin — CHANGES_REQUESTED** [review]

File: components/services/ServiceItem.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import type { FC } from 'react';
+
+import Typography from '@mui/material/Typography';
+import Stack from '@mui/material/Stack';
+
+import Link from '../../components/common/link/Link';
```
[the remark this answers is not in the document]
Comment (InnaLazarenko) [reply components/services/ServiceItem.tsx:6]: '+

File: pages/index.tsx (hunk 1 above)
Comment (InnaLazarenko) [reply pages/index.tsx:21]: '+

File: components/services/Services.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+            What you will get from us
+          </Typography>
+          <Typography align="center" sx={{ marginBottom: { xs: '30px', md: '57px' } }}>
+            Ensure your product is high-performance with a team of QA experts by your side.
+          </Typography>
+        </Box>
+
+        <Grid container spacing={{ xs: 2, md: 3.75 }} columns={12} justifyContent="center">
+          <Grid item xs={12} sm={6} md={4}>
+            <ServiceItem
```
[the remark this answers is not in the document]
Comment (InnaLazarenko) [reply components/services/Services.tsx:46]: '+

File: components/services/ServiceItem.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  link: string;
+  title: string;
+  text: string;
+}
+
+const ServiceItem: FC<ServiceItemProps> = props => {
+  const { icon, link, title, text } = props;
+
+  return (
+    <Link href={link}>
```
[the remark this answers is not in the document]
Comment (InnaLazarenko) [reply components/services/ServiceItem.tsx]: '+

File: components/common/link/NavigationLink.tsx [hunk 5]
```diff
@@ -5,9 +5,9 @@ import { useRouter } from 'next/router';
 
 import MuiLink, { LinkProps } from '@mui/material/Link';
 
-type NavigationLinkProps = LinkProps & NextLinkProps;
+type NavLinkProps = LinkProps & NextLinkProps;
```
Comment (var-bin) [inline components/common/link/NavigationLink.tsx:8]: The main rule of thumb is to follow `<ComponentName>Props` pattern in naming interfaces and types.

File: components/common/link/Link.tsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
+import type { ElementType, FC } from 'react';
+
+import NextLink from 'next/link';
+
+import MuiLink, { LinkProps } from '@mui/material/Link';
+
+export type NavigationLinkProps = LinkProps &  {
```
Comment (var-bin) [inline components/common/link/Link.tsx:7]: Same here.

File: components/common/link/Link.tsx [hunk 7]
```diff
@@ -0,0 +1,27 @@
+import type { ElementType, FC } from 'react';
+
+import NextLink from 'next/link';
+
+import MuiLink, { LinkProps } from '@mui/material/Link';
```
Comment (var-bin) [inline components/common/link/Link.tsx:5]: Please, add `type` keyword. We can use  `LinkProps as MuiLinkProps`

File: components/services/Services.tsx [hunk 8]
```diff
@@ -0,0 +1,78 @@
+import type { FC } from 'react';
+
+import Typography from '@mui/material/Typography';
+import Grid from '@mui/material/Grid';
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+
+import { Service } from '../../models';
```
Comment (var-bin) [inline components/services/Services.tsx:8]: Please, add `type` keyword

**Review by var-bin — CHANGES_REQUESTED** [review]

File: components/common/link/Link.tsx (hunk 7 above)
Comment (InnaLazarenko) [reply components/common/link/Link.tsx]: '+

File: components/common/link/NavigationLink.tsx (hunk 5 above)
Comment (InnaLazarenko) [reply components/common/link/NavigationLink.tsx]: '+

File: components/services/Services.tsx [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+            Ensure your product is high-performance with a team of QA experts by your side.
+          </Typography>
+        </Box>
+
+        <Grid container spacing={{ xs: 2, md: 3.75 }} columns={12} justifyContent="center">
+          {data.map(service => {
+            return (
+              <Grid key={service.id} item xs={12} sm={6} md={4}>
+                <ServiceItem
+                  icon={icons[service.serviceIcon as T]}
```
[the remark this answers is not in the document]
Comment (InnaLazarenko) [reply components/services/Services.tsx]: '+

**Pull request merged**